src/screens/Dashboard/BabyDashboard/BabyDashboard.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import Button from "../../../components/Button/Button";
+import Input from "../../../components/Input/Input";
+import TodoList, { Todo } from "../../../components/TodoList/TodoList";
+import Header from "../../../components/Header/Header";
+import "./BabyDashboard.css";
+
+function BabyDashboard() {
+  const [todos, setTodos] = useState<Todo[]>([
+    { id: 1, text: "Bath Time", completed: false },
+    { id: 2, text: "You need to change diaper", completed: false },
+  ]);
+  const [todoText, setTodoText] = useState<string>("");
+
+  const handleToggle = (id: number) => {
+    const newTodos: Todo[] = todos.map((todo) => {
+      if (todo.id === id) {
+        return { ...todo, completed: !todo.completed };
+      }
+      return todo;
+    });
+    setTodos(newTodos);
+  };
+
+  const handleAddClick = () => {
+    if (!todoText) return;
+    const newTodo: Todo = { id: Date.now(), text: todoText, completed: false };
+    setTodos([...todos, newTodo]);
+  };
+  const handleDeleteClick = () => {
+    const newTodos: Todo[] = todos.filter((todo) => !todo.completed);
+    setTodos(newTodos);
+  };
+
+  const handleInputChange = (value: string) => {
+    setTodoText(value);
+  };
+
+  const itemsToDelete = (): boolean => {
+    for (let todo of todos) {
+      if (todo.completed) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  return (
+    <div className="main-container">
+      <Header headerText="Todo List" />
```

Comment:
Change the header string here to be more descriptive of the actual list, `Todo List` -> `Baby Todos`